task/cloudevent/0.1/cloudevent.yaml

```diff
@@ -33,11 +31,13 @@ spec:
     default: ""
   steps:
   - name: send
-    image: github.com/knative/eventing/cmd/sendevent
+    image: iancoffey/sendevent-89c3de01c503ceb66b676a3e41c265ed@sha256:fc39998cba7ad8e54323b948b56b11b0b235996069349fb74776865fedf182a5
```

Comment:
naive question: is this image not tagged/shipped as part of `knative/eventing` release ? :thinking: 
/cc @matzew

Reply:
>naive question: is this image not tagged/shipped as part of knative/eventing release ? 🤔

Good question, I didnt see an official one on dockerhub at least. Im not sure the best way to handle this image aside from providing the `ko` part in the readme.

Reply:
There is one from openshift `quay.io/openshift-knative/knative-eventing-sendevent:latest` if we do not found anything there and if it is good to use :D

Reply:
so there might be one on `gcr.io` too, let me check

Reply:
Interestingly enough, I didn't find any https://console.cloud.google.com/gcr/images/knative-releases/GLOBAL/knative.dev/eventing/cmd?gcrImageListsize=30 (or https://console.cloud.google.com/gcr/images/knative-releases/GLOBAL/knative.dev/eventing-contrib?gcrImageListsize=30) :sweat_smile:

Reply:
I tried looking, did not find any. Hope you find one